src/projections/analysis/IntervalData.java

```diff
@@ -237,6 +256,34 @@ public int getNumIntervals() {
     public double getIntervalSize() {
 	return intervalSize;
     }
+		public double[][] getMsg_count() 
+    {
+	return msg_count;
+    }
+
+	public double[][] getMsg_size() 
+    {
+	return msg_size;
+    }
+	public double[][] getMsg_recv_count() 
+    {
+	return msg_recv_count;
+    }
+
+	public double[][] getMsg_recv_size() 
+    {
+	return msg_recv_size;
+    }
+	public double[][] getMsg_recv_count_ext() 
+    {
+	return msg_recv_count_ext;
+    }
+
+	public double[][] getMsg_recv_size_ext() 
+    {
+	return msg_recv_size_ext;
+    }
```

Comment:
I think this can all be done via the existing `getData` function, see other comments.